hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/failure/CustomExceptionalHaltReason.java

```diff
@@ -76,6 +76,8 @@ public static ResponseCodeEnum statusFor(@NonNull final ExceptionalHaltReason re
             return ResponseCodeEnum.MAX_CHILD_RECORDS_EXCEEDED;
         } else if (reason == CustomExceptionalHaltReason.INVALID_CONTRACT_ID) {
             return ResponseCodeEnum.INVALID_CONTRACT_ID;
+        } else if (reason == CustomExceptionalHaltReason.INVALID_FEE_SUBMITTED) {
```

Comment:
At some point in time, as part of a general code cleanup, this if-ladder should be replaced by something more directly readable.  E.g., each enum `CustomExceptionalHaltReason` could have a field which is its mapped status.  Then the mapped status would be, syntactically, right next to the declaration of its enum value, and thus, the whole thing would be easier to read.  

SImilarly for `errorMessageFor`.  That one is not too complicated, but still, data-driven is better, IMO.

For the future.  (Perhaps you could add a `// FUTURE:` comment around here so we can find it later when we pass through the contract service looking for cleanups.)